io/include/pcl/compression/color_coding.h

```diff
@@ -72,8 +72,7 @@ class ColorCoding
   /** \brief Empty class constructor. */
   virtual
   ~ColorCoding ()
-  {
-  }
+  = default;
```

Comment:
Other places also have the `= default` split on 2 lines

Reply:
Caught a few more, hope they're the last